operations/exports.md

```diff
@@ -0,0 +1,158 @@
+| `EntityType` | string [Entity type](#entity-type) | optional | Return only exports with the provided [Entity type](#entity-type). |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of message data returned (using cursor pagination). |
+
+### Response
+
+```json
+{
+    "Exports": [
+        {
+            "Id": "3fa85f64-5717-4562-b3fd-2c963f66afa6",
+            "Status": "Success",
+            "EntityType": "OrderItem",
+            "ExpiresUtc": "2023-10-26T11:42:28Z",
+            "Files": [
+                {
+                    "Url": "https://example.com/exports/3fa85f64-5717-4562-b3fd-2c963f66afa6-1.jsonl?example=signature",
+                    "SizeInBytes": 1215279
+                },
+                {
+                    "Url": "https://example.com/exports/3fa85f64-5717-4562-b3fd-2c963f66afa6-2.jsonl?example=signature",
+                    "SizeInBytes": 1398362
+                },
+            ]
+        },
+        {
+            "Id": "09708665-0e31-4b23-b337-b0a000be0df0",
+            "Status": "Pending",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": null
+        },
+        {
+            "Id": "706dc6d5-9511-4751-825e-538ce99da2ce",
+            "Status": "Processing",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": null
+        },
+        {
+            "Id": "706dc6d5-9511-4751-825e-538ce99da2ce",
+            "Status": "Expired",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": "2023-10-24T14:12:30Z"
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Exports` | array of [Exports](#export) | required | Requested exports. |
+
+#### Export
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the export. |
+| `Status` | string [Export status](#export-status) | required | Current status of the export. |
+| `EntityType` | string [Entity type](#entity-type) | required | Type of exported entities. |
+| `Files` | array of [Exported files](#exported-file) | required | Files with exported data, if available. |
+| `ExpiresUtc` | string | optional | Expiration date and time of the export in UTC timezone in ISO 8601 format. After this time the [exported files](#exported-file) may no longer be available for download. |
+
+#### Export status
+
+* `Pending` - export was created and didn't start yet
+* `Processing` - export process has started
+* `Success` - export has been successfully finished and [exported files](#exported-file) are available for download
+* `Expired` - export has expired and exported files are no longer available
+* `Failed` - export has failed and exported files are not available; you can try creating a new export
```

Comment:
Is there a chance we would add another status? 🤔 If so, we should mark it as extendable.

```suggestion
* `Pending` - export was created and didn't start yet
* `Processing` - export process has started
* `Success` - export has been successfully finished and [exported files](#exported-file) are available for download
* `Expired` - export has expired and exported files are no longer available
* `Failed` - export has failed and exported files are not available; you can try creating a new export
* ...
```

Reply:
I find it very unlikely we'll need another status (in fact, `Failed` is currently unused) and we're still in beta so we can change it as we desire.